packages/server/postgres/migrations/1650768593875_add-AzureDevOpsDimensionFieldMap.ts

```diff
@@ -0,0 +1,28 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  CREATE TABLE IF NOT EXISTS "AzureDevOpsDimensionFieldMap" (
+    "id" SERIAL,
```

Comment:
+1 could change to `"id" SERIAL UNIQUE` to ensure it's a unique id

Reply:
@nickoferrall  - yes.  The additional PR for #6512 will resolve the issue you are referring to.

Reply:
I'll change it from "Issues..." to "Work Items..." on the PR for #6512.

For the issue above with the hypermedia url vs rest url, I was going to create an issue as a bug and assign to me.  I have been holding off creating 2 issues because I was going to label them as bugs, but didn't want to create a bug until it was officially in the master branch.  I plan on creating them, assigning them to me, and fixing them.